### DISCUSSION TITLE
ReferenceError: DEDICATED_COMPRESSOR_4KB is not defined

### DISCUSSION THREAD

**Discussion by Tuner32** [uNetworking/uWebSockets.js#609] [Q&A]
What is wrong with this? I use normal  code and I just changed for nothing except SSLApp to App. it wor wll if I remove compressor option It work well. I used example code

code: 
/* Non-SSL is simply App() */
require('uWebSockets.js').App().ws('/*', {

  /* There are many common helper features */
  idleTimeout: 30,
  maxBackpressure: 1024,
  maxPayloadLength: 512,
  compression: DEDICATED_COMPRESSOR_3KB,

  /* For brevity we skip the other events (upgrade, open, ping, pong, close) */
  message: (ws, message, isBinary) => {
    /* You can do app.publish('sensors/home/temperature', '22C') kind of pub/sub as well */
    
    /* Here we echo the message back, using compression if available */
    let ok = ws.send(message, isBinary, true);
  }
  
}).get('/*', (res, req) => {

  /* It does Http as well */
  res.writeStatus('200 OK').writeHeader('IsExample', 'Yes').end('Hello there!');
  
}).listen(9001, (listenSocket) => {

  if (listenSocket) {
    console.log('Listening to port 9001');
  }
  
});

I used this on node 14.17.6
and it on window 10 1803

**Comment by kunal-mandalia** [comment]
Importing constants from package worked for me, e.g.:

```
const { DEDICATED_COMPRESSOR_3KB } = require('uWebSockets.js');
```
